src/en/jaas-cli.md

```diff
@@ -0,0 +1,62 @@
+Title: Using JAAS from the command line
+
+# Using JAAS from the command line
+
+To use JAAS from the CLI requires that you install Juju on a local machine and
+connect that Juju install with your JAAS account.
+
+## Install Juju
+
+Follow [these instructions][installjuju] to install Juju on your local machine.
+
+## Register or login to JAAS from Juju
+
+If you are using Juju 2.1, use:
+
+```bash
+juju register jimm.jujucharms.com
+```
+
+If you are using Juju 2.2, use:
+	
+```bash
+juju login JAAS
```

Comment:
Figured it out. The answer is that we should leave it as originally written, 'jaas'. I introduced the error during a search/replace updating all the JaaS references to JAAS and accidentally "fixed" this one, too. Reverting to all lower case.